diff --git a/lib/cmake/LIEF/LIEFExport-static.cmake b/lib/cmake/LIEF/LIEFExport-static.cmake
index c36f3a6..c382edc 100644
--- a/lib/cmake/LIEF/LIEFExport-static.cmake
+++ b/lib/cmake/LIEF/LIEFExport-static.cmake
@@ -47,10 +47,7 @@ unset(_cmake_expected_targets)


 # Compute the installation prefix relative to this file.
-get_filename_component(_IMPORT_PREFIX "${CMAKE_CURRENT_LIST_FILE}" PATH)
-get_filename_component(_IMPORT_PREFIX "${_IMPORT_PREFIX}" PATH)
-get_filename_component(_IMPORT_PREFIX "${_IMPORT_PREFIX}" PATH)
-get_filename_component(_IMPORT_PREFIX "${_IMPORT_PREFIX}" PATH)
+set(_IMPORT_PREFIX "${CMAKE_CURRENT_LIST_DIR}/../..")
 if(_IMPORT_PREFIX STREQUAL "/")
   set(_IMPORT_PREFIX "")
 endif()
