diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9d85e165..264589a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -213,7 +213,9 @@ include( cmake/BuildOptions.cmake )
 include( cmake/CompilerOptions.cmake )
 
 # dependencies
-include( cmake/Dependencies.cmake )
+#include( cmake/Dependencies.cmake )
+find_package(7zip CONFIG REQUIRED)
+add_library(7-zip ALIAS 7zip::7zip)
 
 if( BIT7Z_USE_PCH AND CMAKE_VERSION VERSION_GREATER_EQUAL "3.16.0" )
     target_precompile_headers( ${LIB_TARGET}
@@ -245,7 +247,7 @@ target_link_libraries( ${LIB_TARGET} PRIVATE 7-zip )
 #  - empty library for modern compilers (no extra linking required).
 #  - libstdc++fs/libc++fs for older versions of libstdc++/libc++.
 #  - ghc::filesystem for even older libraries supporting only C++11/C++14 standards.
-target_link_libraries( ${LIB_TARGET} PRIVATE filesystem_lib )
+#target_link_libraries( ${LIB_TARGET} PRIVATE filesystem_lib )
 if( NOT ghc_filesystem_ADDED AND BIT7Z_USE_PCH AND CMAKE_VERSION VERSION_GREATER_EQUAL "3.16.0" )
     target_precompile_headers( ${LIB_TARGET} PRIVATE <filesystem> )
 endif()
