diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7b0d386..c6186ad 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,6 @@ if(POLICY CMP0167)
 endif()
 project(libloot)
 include(ExternalProject)
-include(FetchContent)
 include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)

@@ -49,7 +48,11 @@ configure_file("${CMAKE_SOURCE_DIR}/src/api/loot_version.cpp.in" "${CMAKE_BINARY
 set(Boost_USE_STATIC_LIBS ON)
 set(Boost_USE_MULTITHREADED ON)
 set(Boost_USE_STATIC_RUNTIME OFF)
-find_package(Boost REQUIRED CONFIG)
+find_package(Boost REQUIRED CONFIG COMPONENTS algorithm container crc graph unordered)
+
+find_package(fmt CONFIG REQUIRED)
+find_package(spdlog CONFIG REQUIRED)
+find_package(yaml-cpp CONFIG REQUIRED)

 if(NOT DEFINED RUST_TARGET AND WIN32)
     if(MSVC)
@@ -156,41 +159,6 @@ else()
     set(LCI_LIBRARIES ${LCI_LIBRARIES} dl)
 endif()

-FetchContent_Declare(
-    fmt
-    URL "https://github.com/fmtlib/fmt/archive/refs/tags/11.2.0.tar.gz"
-    URL_HASH "SHA256=bc23066d87ab3168f27cef3e97d545fa63314f5c79df5ea444d41d56f962c6af"
-    FIND_PACKAGE_ARGS)
-
-set(SPDLOG_FMT_EXTERNAL ON)
-FetchContent_Declare(
-    spdlog
-    URL "https://github.com/gabime/spdlog/archive/v1.15.3.tar.gz"
-    URL_HASH "SHA256=15a04e69c222eb6c01094b5c7ff8a249b36bb22788d72519646fb85feb267e67"
-    FIND_PACKAGE_ARGS)
-
-set(YAML_MSVC_SHARED_RT ON)
-set(YAML_CPP_BUILD_CONTRIB OFF)
-set(YAML_CPP_BUILD_TOOLS OFF)
-set(YAML_BUILD_SHARED_LIBS OFF)
-set(YAML_CPP_BUILD_TESTS OFF)
-# Suppress additional targets added by CTest, which is pulled in by yaml-cpp.
-set_property(GLOBAL PROPERTY CTEST_TARGETS_ADDED 1)
-FetchContent_Declare(
-    yaml-cpp
-    URL "https://github.com/loot/yaml-cpp/archive/0.8.0+merge-key-support.3.tar.gz"
-    URL_HASH "SHA256=e2067f1ab8f658aeb8a8795fd0e06a753b4150b9352cae047280e659078dd44f")
-
-# Set BUILD_SHARED_LIBS=OFF to prevent fmt and spdlog from being built as shared
-# libraries.
-set(LIBLOOT_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
-set(BUILD_SHARED_LIBS OFF)
-
-FetchContent_MakeAvailable(fmt spdlog yaml-cpp)
-
-# Restore the original value of BUILD_SHARED_LIBS.
-set(BUILD_SHARED_LIBS ${LIBLOOT_BUILD_SHARED_LIBS})
-
 ##############################
 # General Settings
 ##############################
@@ -312,7 +280,11 @@ add_dependencies(loot
     libloadorder
     loot-condition-interpreter)
 target_link_libraries(loot PRIVATE
-    Boost::headers
+    Boost::algorithm
+    Boost::container
+    Boost::crc
+    Boost::graph
+    Boost::unordered
     ${ESPLUGIN_LIBRARIES}
     ${LIBLOADORDER_LIBRARIES}
     ${LCI_LIBRARIES}
